Include/pystate.h

```diff
@@ -75,18 +75,39 @@ typedef struct {
     wchar_t *dll_path;      /* Windows DLL path */
 #endif
 
+    /* If greater than 0, enable isolated mode sys.path contains
+       neither the script's directory nor the user's site-packages directory.
+       Ignored if set to -1.
```

Comment:
What if it's set to 0? Is it ignored in that case as well? (the comment covers >1 and -1 but not 0)

Reply:
I updated the doc.